view/perspective/app.js

```diff
@@ -249,36 +249,41 @@ function handleLibraryFiles(res) {
  */
 function getFilterQuery(p) {
   let q = '?';
+
   if (p.aspectFilter && p.aspectFilter.length) {
-    const sign = p.aspectFilterType === 'INCLUDE' ? '=' : '=-';
-    q += 'aspect' + sign + p.aspectFilter.join();
+    const sign = p.aspectFilterType === 'INCLUDE' ? '' : '-';
+    q += 'aspect' + '=' + sign + 
+        p.aspectFilter.join().replace(/,/g, ',' + sign);
```

Comment:
Does it really need the sign before each one?
wouldn't `...?aspect=-A,B,C` set up an EXCLUDE filter for all three aspects without having to put the minus sign before each one `...?aspect=-A,-B,-C`?